<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NetProxyPlugin</class>
 <widget class="QFrame" name="NetProxyPlugin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>579</width>
    <height>83</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>83</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QGroupBox" name="m_gb_netproxy">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Net Proxy</string>
     </property>
     <widget class="QPushButton" name="m_bt_unload">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Unload</string>
      </property>
     </widget>
     <widget class="QPushButton" name="m_bt_settings">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>30</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Settings</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="m_cb_udp">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>24</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>UDP</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_led_udp_enable">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>24</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/images/led-circle-grey-md.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="m_led_udp_tx">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>24</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/images/led-circle-grey-md.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="m_lbl_udp_tx">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>24</y>
        <width>21</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Tx:</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_led_udp_rx">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>24</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/images/led-circle-grey-md.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="m_lbl_udp_rx">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>24</y>
        <width>21</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Rx:</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_lbl_udp_en">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>24</y>
        <width>21</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>En:</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_lbl_tcp_rx">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>44</y>
        <width>21</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Rx:</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_led_tcp_tx">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>44</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/images/led-circle-grey-md.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="m_lbl_tcp_en">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>44</y>
        <width>21</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>En:</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_led_tcp_enable">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>44</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/images/led-circle-grey-md.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="m_lbl_tcp_tx">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>44</y>
        <width>21</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Tx:</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_led_tcp_rx">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>44</y>
        <width>20</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resources.qrc">:/images/led-circle-grey-md.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="m_cb_tcp">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>44</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>TCP</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_lbl_udp_status">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>24</y>
        <width>181</width>
        <height>17</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>52</red>
            <green>101</green>
            <blue>164</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>52</red>
            <green>101</green>
            <blue>164</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>52</red>
            <green>101</green>
            <blue>164</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>52</red>
            <green>101</green>
            <blue>164</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>190</red>
            <green>190</green>
            <blue>190</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>190</red>
            <green>190</green>
            <blue>190</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="text">
       <string>udp status</string>
      </property>
     </widget>
     <widget class="QLabel" name="m_lbl_tcp_status">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>44</y>
        <width>181</width>
        <height>17</height>
       </rect>
      </property>
      <property name="palette">
       <palette>
        <active>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>52</red>
            <green>101</green>
            <blue>164</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>52</red>
            <green>101</green>
            <blue>164</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>52</red>
            <green>101</green>
            <blue>164</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>52</red>
            <green>101</green>
            <blue>164</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="WindowText">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>190</red>
            <green>190</green>
            <blue>190</blue>
           </color>
          </brush>
         </colorrole>
         <colorrole role="Text">
          <brush brushstyle="SolidPattern">
           <color alpha="255">
            <red>190</red>
            <green>190</green>
            <blue>190</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="text">
       <string>tcp status</string>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
